### PR TITLE
feat(notifications) Use our own component

### PR DESCRIPTION
**Pull request by robertpin**
Use our component instead of atlaskit
Implement redesign
Add unmount animation
Change mount animation from external lib to CSS
Remove @atlaskit/flag
Remove react-transition-group
Add unmount animation on native
Remove AbstractNotification

https://user-images.githubusercontent.com/11474153/224712781-782be7b1-f056-4b56-9b06-fe712586f7e0.mov

### CODE REVIEW

File: react/features/notifications/components/native/NotificationsContainer.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { SafeAreaView } from 'react-native-safe-area-context';
 import { connect } from '../../../base/redux';
```
Comment (Calinteodor) [inline react/features/notifications/components/native/NotificationsContainer.js:7]: Can we turn this into .tsx, too?

Comment (robertpin) [reply react/features/notifications/components/native/NotificationsContainer.js:7]: Kinda outside the scope

Comment (Calinteodor) [reply react/features/notifications/components/native/NotificationsContainer.js:7]: It could be done in a separate commit.

Comment (robertpin) [reply react/features/notifications/components/native/NotificationsContainer.js:7]: In a separate PR

**Review by Calinteodor — APPROVED** [review]

**Pull request merged**